flink-connectors/flink-jdbc/src/main/java/org/apache/flink/api/java/io/jdbc/JDBCOutputFormat.java

```diff
@@ -58,6 +68,11 @@
 
 	private int[] typesArray;
 
+	private Meter batchLimitReachedMeter;
+	private Meter flushMeter;
```

Comment:
These could be initialized in the constructor and made final.

Reply:
they need the runtime to be initialised though